code/public/index.html

```diff
@@ -13,7 +12,18 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <meta property="og:title" content="On my list: Your new todo-app!" />
+    <meta
+      property="og:description"
+      content="Keep track of all your todos! Make a list of your tasks, add and delete as you please. An app done by a Technigo student, built with Redux toolkit."
+    />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="#" />
+    <meta
+      property="og:image"
+      content="https://maria-petersson-todo.netlify.app/img/todo_og.png"
+    />
```

Comment:
Really complete meta tags! I saw them in action in Slack and LinkedIn, and they work well.